src/VisualStudio/Core/Test/InheritanceMargin/InheritanceMarginViewModelTests.vb

```diff
@@ -71,8 +71,9 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.InheritanceMargin
                     Assert.True(acutalLineToTagDictionary.ContainsKey(lineNumber))
 
                     Dim acutalTag = acutalLineToTagDictionary(lineNumber)
+                    ' Editor TestView zoom level is 100 based.
                     Dim actualViewModel = InheritanceMarginViewModel.Create(
-                        classificationTypeMap, classificationFormatMap.GetClassificationFormatMap("tooltip"), acutalTag, 1)
+                        classificationTypeMap, classificationFormatMap.GetClassificationFormatMap("tooltip"), acutalTag, 100)
```

Comment:
All the unit tests below are expecting a ViewModel that has 1 scaleFactor, change the value here to be 100 to match TextView.ZoomLevel, it will make sure the conversion is not missed